3scale/3scale-operator
# THREESCALE-10389 Migrate backend DeploymentConfigs to Deployments

# Issue link
JIRA: [THREESCALE-10389](https://issues.redhat.com/browse/THREESCALE-10389)

# What
This PR replaces the `backend-cron`, `backend-listener`, and `backend-worker` DeploymentConfigs with Deployments.

# Verification steps
1. Provision a cluster
2. Install 3scale locally using master
3. Checkout this PR
4. Re-run 3scale locally
5. Verify that the new `backend-cron`, `backend-listener`, and `backend-worker` Deployments are created
6. Verify that the `backend-cron`, `backend-listener`, and `backend-worker` DeploymentConfigs are deleted once the Deployments report healthy
